test/select-list-view.test.js

```diff
@@ -610,19 +610,19 @@ describe('SelectListView', () => {
       assert.equal(selectListView.element.querySelectorAll('.item').length, 100)
       assert.equal(selectListView.element.querySelectorAll('.item.visible').length, 5)
       selectListView.element.style.overflowY = 'auto'
-      selectListView.element.style.height = '150px'
+      selectListView.element.style.height = '150px' // 5 items height
 
       containerNode.appendChild(selectListView.element)
 
       const children = selectListView.refs.items.children
       children[20].scrollIntoViewIfNeeded()
-      await assertItemsBecomeVisible([17, 18, 19, 20, 21, 22, 23])
+      await assertItemsBecomeVisible([18, 19, 20, 21, 22])
```

Comment:
----------------------
`17` and `23` here was "BecomeVisible" because by `intersectionRatio === 0` not by `intersectionRatio > 0`, even in non-problematic Atom-v1.24.0.
So this is the spec created based on incorrect behavior. Now fixed incorrectly behavior, so need to fix spec too.
Just when I implemented it in Atom-v1.24.0, I didn't noticed this incorrectness.

```
[
  17, # intersectionRatio == 0
  18, # intersectionRatio == 1
  19, # intersectionRatio == 1
  20, # intersectionRatio == 1
  21, # intersectionRatio == 1
  22, # intersectionRatio == 1
  23, # intersectionRatio == 0
]
```

To be precise
- Atom-v1.24.0(`Chrome  : 56.0.2924.87`) affect this bug, but not severe, it's just report border TWO invisible elements as `intersectionRatio === 0`.
- Atom-v1.25.0-beta2(`Chrome: 58.0.3029.110`) affect this bug severely, it's report ALL elements as `intersectionRatio === 0`.

I think this change comes from chrome version change by electron version up.

Reply:
Thanks for the explanation!  Makes sense to me.